aws_kube_params.inc

```diff
@@ -2,9 +2,9 @@ export KUBERNETES_PROVIDER=aws
 export CLOUD_PROVIDER=${KUBERNETES_PROVIDER}
 export KUBE_AWS_INSTANCE_PREFIX=arktos
 
-export KUBE_AWS_ZONE=us-east-2b
-export VPC_ID=vpc-0bb8ff51fd37c59b1
-export SUBNET_ID=subnet-00550372a5041fc32
+export KUBE_AWS_ZONE=us-east-1a
+export VPC_ID=vpc-2d43fc57
```

Comment:
try to use VPC_NAME and SUBNET_NAME to replace VPC_ID, SUBNET_ID

Reply:
The feature I'm working on doesn't care vpcid. This request is a feature request in original aws kube-up feature. I can try to work on it later.